modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/components/_tables.scss

```diff
@@ -0,0 +1,109 @@
+.table-list.table-lg {
+    --table-cell-padding-y: var(--table-lg-cell-padding-y);
+    --table-cell-padding-x: var(--table-lg-cell-padding-x);
+    --table-cell-min-height: var(--table-lg-cell-min-height);
+    --table-font-size: var(--table-lg-header-font-size);
+    --table-font-weight: var(--table-lg-font-weight);
+    --table-line-height: var(--table-lg-line-height);
+}
+
+.table-list.table-sm {
+    --table-cell-padding-y: var(--table-sm-cell-padding-y);
+    --table-cell-padding-x: var(--table-sm-cell-padding-x);
+    --table-cell-min-height: var(--table-sm-cell-min-height);
+    --table-font-size: var(--table-sm-header-font-size);
+    --table-font-weight: var(--table-sm-font-weight);
+    --table-line-height: var(--table-sm-line-height);
+}
+
+.table-list.table-lg th {
+    --table-header-cell-padding-y: var(--table-lg-header-cell-padding-y);
+    --table-header-cell-padding-x: var(--table-lg-header-cell-padding-x);
+    --table-header-cell-min-height: var(--table-lg-header-cell-min-height);
+    --table-header-font-size: var(--table-lg-header-header-font-size);
+    --table-header-font-weight: var(--table-lg-header-font-weight);
+    --table-header-line-height: var(--table-lg-header-line-height);
+}
+
+.table-list.table-sm th {
+    --table-header-cell-padding-y: var(--table-sm-header-cell-padding-y);
+    --table-header-cell-padding-x: var(--table-sm-header-cell-padding-x);
+    --table-header-cell-min-height: var(--table-sm-header-cell-min-height);
+    --table-header-font-size: var(--table-sm-header-header-font-size);
+    --table-header-font-weight: var(--table-sm-header-font-weight);
+    --table-header-line-height: var(--table-sm-header-line-height);
+}
+
+th {
+    --table-cell-padding-y: var(--table-header-cell-padding-y);
+    --table-cell-padding-x: var(--table-header-cell-padding-x);
+    --table-cell-min-height: var(--table-header-cell-min-height);
+    --table-font-size: var(--table-header-font-size); 
+    --table-font-weight: var(--table-header-font-weight);
+    --table-line-height: var(--table-header-line-height);
+}
+
```

Comment:
The majority of the css below here, should still be reworked and simplified. Some can use additional clay sass variables and some just needs to have less css selectors applied to it. I'll share a branch with you that has some ideas for how this could be done.

Reply:
Fixed